target/linux/ath79/dts/qca9531_ap147_010_reference_board.dtsi

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	model = "Atheros AP147-010 reference board";
```

Comment:
@blocktrron Do we use model strings in DTSI files?

Reply:
I removed it to be consistent with other dtsi